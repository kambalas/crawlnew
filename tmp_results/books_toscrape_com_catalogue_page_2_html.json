[
  {
    "name": "In Her Wake",
    "price": "12.84",
    "error": false
  },
  {
    "name": "How Music Works",
    "price": "37.32",
    "error": false
  },
  {
    "name": "Foolproof Preserving: A Guide to Small Batch Jams, Jellies, Pickles, Condiments, and More: A Foolproof Guide to Making Small Batch Jams, Jellies, Pickles, Condiments, and More",
    "price": "30.52",
    "error": false
  },
  {
    "name": "Chase Me (Paris Nights #2)",
    "price": "25.27",
    "error": false
  },
  {
    "name": "Black Dust",
    "price": "34.53",
    "error": false
  },
  {
    "name": "Birdsong: A Story in Pictures",
    "price": "54.64",
    "error": false
  },
  {
    "name": "America's Cradle of Quarterbacks: Western Pennsylvania's Football Factory from Johnny Unitas to Joe Montana",
    "price": "22.50",
    "error": false
  },
  {
    "name": "Aladdin and His Wonderful Lamp",
    "price": "53.13",
    "error": false
  },
  {
    "name": "Worlds Elsewhere: Journeys Around Shakespeare's Globe",
    "price": "40.30",
    "error": false
  },
  {
    "name": "Wall and Piece",
    "price": "44.18",
    "error": false
  },
  {
    "name": "The Four Agreements: A Practical Guide to Personal Freedom",
    "price": "17.66",
    "error": false
  },
  {
    "name": "The Five Love Languages: How to Express Heartfelt Commitment to Your Mate",
    "price": "31.05",
    "error": false
  },
  {
    "name": "The Elephant Tree",
    "price": "23.82",
    "error": false
  },
  {
    "name": "The Bear and the Piano",
    "price": "36.89",
    "error": false
  },
  {
    "name": "Sophie's World",
    "price": "15.94",
    "error": false
  },
  {
    "name": "Penny Maybe",
    "price": "33.29",
    "error": false
  },
  {
    "name": "Maude (1883-1993):She Grew Up with the country",
    "price": "18.02",
    "error": false
  },
  {
    "name": "In a Dark, Dark Wood",
    "price": "19.63",
    "error": false
  },
  {
    "name": "Behind Closed Doors",
    "price": "52.22",
    "error": false
  },
  {
    "name": "You can't bury them all: Poems",
    "price": "33.63",
    "error": false
  }
]