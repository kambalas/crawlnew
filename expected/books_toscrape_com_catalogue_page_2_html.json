[
  {
    "name": "In Her Wake",
    "price": "12.84",
    "error": false
  },
  {
    "name": "How Music Works",
    "price": "37.32",
    "error": false
  },
  {
    "name": "Foolproof Preserving: A Guide to Small Batch Jams, Jellies, Pickles, Condiments, and More: A Foolproof Guide to Making Small Batch Jams, Jellies, Pickles, Condiments, and More",
    "price": "30.52",
    "error": false
  },
  {
    "name": "Chase Me (Paris Nights ...",
    "price": "25.27",
    "error": false
  },
  {
    "name": "Black Dust",
    "price": "34.53",
    "error": false
  },
  {
    "name": "Birdsong: A Story in ...",
    "price": "54.64",
    "error": false
  },
  {
    "name": "America's Cradle of Quarterbacks: Western Pennsylvania's Football Factory from Johnny Unitas to Joe Montana",
    "price": "22.50",
    "error": false
  },
  {
    "name": "Aladdin and His Wonderful ...",
    "price": "53.13",
    "error": false
  },
  {
    "name": "Worlds Elsewhere: Journeys Around ...",
    "price": "40.30",
    "error": false
  },
  {
    "name": "Wall and Piece",
    "price": "44.18",
    "error": false
  },
  {
    "name": "The Four Agreements: A ...",
    "price": "17.66",
    "error": false
  },
  {
    "name": "The Five Love Languages: ...",
    "price": "31.05",
    "error": false
  },
  {
    "name": "The Elephant Tree",
    "price": "23.82",
    "error": false
  },
  {
    "name": "The Bear and the ...",
    "price": "36.89",
    "error": false
  },
  {
    "name": "Sophieâ€™s World",
    "price": "15.94",
    "error": false
  },
  {
    "name": "Penny Maybe",
    "price": "33.29",
    "error": false
  },
  {
    "name": "Maude (1883-1993):She Grew Up ...",
    "price": "18.02",
    "error": false
  },
  {
    "name": "In a Dark, Dark Wood",
    "price": "19.63",
    "error": false
  },
  {
    "name": "Behind Closed Doors",
    "price": "52.22",
    "error": false
  },
  {
    "name": "You can't bury them all: Poems",
    "price": "33.63",
    "error": false
  }
]